h5grove/utils.py

```diff
@@ -175,6 +180,28 @@ def parse_bool_arg(query_arg: Union[str, None], fallback: bool) -> bool:
     return query_arg.lower() != "false"
 
 
+def parse_link_resolution_arg(
+    raw_query_arg: Union[str, None], fallback: LinkResolution
+) -> LinkResolution:
+    if raw_query_arg is None:
+        return fallback
+
+    query_arg = raw_query_arg.lower()
+
+    if query_arg == "true" or query_arg == LinkResolution.ALL.value:
```

Comment:
Checking for `true` and `false` make the change non-breaking for requests that specified the `resolve_links` query arg

Reply:
The test can also be written (but that's a matter of taste, up to you):
```suggestion
    if query_arg in ("true", LinkResolution.ALL.value):
```

Reply:
Maybe add a comment to make clear `"true"` and `"false"` are for backward compatibility.